docs/developers_guide/ocean/framework.md

```diff
@@ -422,3 +422,13 @@ density, which is horizontally constant and increases with depth.
 The {py:func}`polaris.ocean.rpe.compute_rpe()` is used to compute the RPE as
 a function of time in a series of one or more output files.  The RPE is stored
 in `rpe.csv` and also returned as a numpy array for plotting and analysis.
+
+## Visualization
```

Comment:
Yes, good point about `nCells` by `nVertLevels`.  I'll add that.

I will also change 3D field to something else.  I don't think it is clearer to call it 2D because we often use "2D" as a shorthand for horizontal and "3D" as a shorthand for "horizontal and vertical".

Reply:
Fixed!